src/components/local-scene-sidebar/local-scene-sidebar-selectors.js

```diff
@@ -17,6 +17,24 @@ const getDescription = createSelector(selectCountryData, countryData => {
   return countryData.sentence;
 })
 
+const getHasPriority = createSelector(selectCountryData, countryData => {
+  if (!countryData) return null;
+  return countryData.has_priority === 1;
+})
+
+const getPriorityAreasSentence = createSelector([selectCountryData, getHasPriority], (countryData, hasPriority) => {
+  if (!countryData) return null;
+  return hasPriority ?
+  `The brightly colored map layer presents one possible configuration
```

Comment:
I'm not sure if this is a bit hidden inside a selector and should belong to the component. But it's not a big deal

Reply:
It was inside the component before, but since it is used also on the pdf report being in the selector allows us to update the text only in one place and use it on both components.